Frameworks/CanvasKeymaster/CanvasKeymaster/CanvasKeymaster.m

```diff
@@ -122,6 +122,12 @@ - (void)loginWithMobileVerifyDetails:(NSDictionary *)details {
     }];
 }
 
+- (void)setupWithClient:(CKIClient *)client {
+    [self setCurrentClient:client];
+    [[FXKeychain sharedKeychain] clearKeychain];
+    [self->_subjectForClientLogin sendNext:client];
```

Comment:
Do we need the `->`? `[self.subjectForClientLogin sendNext:client];`? I'm sure this works so I won't block on this.

Reply:
I was copying what is done when you log in through keymaster.